notificationsManager/Operators/OperatorsService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace NotificationsManager\Operators;
+
+use Exception;
+use NotificationsManager\Operators\Database\Entities\Operator;
+use NotificationsManager\Operators\Repositories\OperatorRepository;
+
+class OperatorsService
+{
+    private OperatorRepository $operatorRepository;
+    public function __construct(OperatorRepository $operatorRepository)
+    {
+        $this->operatorRepository = $operatorRepository;
+    }
```

Comment:
Desde PHP 8.2 puedes hacer esto así:
```suggestion
    public function __construct(private readonly OperatorRepository $operatorRepository)
    {
    }
```